data-upgrade-news/src/main/resources/conf/portal/configuration.xml

```diff
@@ -254,6 +254,50 @@
         </value-param>
       </init-params>
     </component-plugin>
+
+    <component-plugin>
+      <name>PublishedNewsImagesPermissionsUpgradePlugin</name>
+      <set-method>addUpgradePlugin</set-method>
+      <type>org.exoplatform.news.upgrade.jcr.PublishedNewsImagesPermissionsUpgradePlugin</type>
+      <description>Update published news images permissions</description>
+      <init-params>
+        <value-param>
+          <name>product.group.id</name>
+          <description>The groupId of the product</description>
+          <value>org.exoplatform.platform</value>
+        </value-param>
+        <value-param>
+          <name>platform.users.group.identity</name>
+          <description>The identity of the platform users group</description>
+          <value>*:/platform/users</value>
+        </value-param>
+        <value-param>
+          <name>permissions</name>
+          <description>Permissions to be added to the published news images</description>
+          <value>read</value>
+        </value-param>
```

Comment:
I am not sure if this UP will be reused, is there a need to parameterize those properties ?